app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsScreen.kt

```diff
@@ -0,0 +1,240 @@
+                message = dialogState.message(),
+                onDismissRequest = handler.onDismissDialog,
+                confirmButtonText = confirmButtonText,
+                dismissButtonText = dismissButtonText,
+                onConfirmClick = handler.onConfirmGetStarted,
+                onDismissClick = handler.onDismissDialog,
+            )
+        }
+
+        ImportLoginsState.DialogState.ImportLater -> {
+            BitwardenTwoButtonDialog(
+                title = dialogState.title(),
+                message = dialogState.message(),
+                onDismissRequest = handler.onDismissDialog,
+                confirmButtonText = confirmButtonText,
+                dismissButtonText = dismissButtonText,
+                onConfirmClick = handler.onConfirmImportLater,
+                onDismissClick = handler.onDismissDialog,
+            )
+        }
+
+        null -> Unit
+    }
+}
+
+@Composable
+private fun ImportLoginsContent(
+    onGetStartedClick: () -> Unit,
+    onImportLaterClick: () -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState())
+            .padding(top = 24.dp)
+            .navigationBarsPadding(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Image(
+            painter = rememberVectorPainter(R.drawable.img_import_logins),
+            contentDescription = null,
+            modifier = Modifier
+                .standardHorizontalMargin()
+                .size(124.dp),
+        )
+        Spacer(Modifier.height(24.dp))
+        Text(
+            text = stringResource(R.string.give_your_vault_a_head_start),
+            style = BitwardenTheme.typography.titleMedium,
+            color = BitwardenTheme.colorScheme.text.primary,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(12.dp))
+        Text(
+            text = stringResource(
+                R.string.from_your_computer_follow_these_instructions_to_export_saved_passwords,
+            ),
+            style = BitwardenTheme.typography.bodyMedium,
+            color = BitwardenTheme.colorScheme.text.primary,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(24.dp))
+        BitwardenFilledButton(
+            label = stringResource(R.string.get_started),
+            onClick = onGetStartedClick,
+            modifier = Modifier
+                .standardHorizontalMargin()
+                .fillMaxWidth(),
+        )
+        Spacer(Modifier.height(12.dp))
+        BitwardenOutlinedButton(
+            label = stringResource(R.string.import_logins_later),
+            onClick = onImportLaterClick,
+            modifier = Modifier
+                .standardHorizontalMargin()
+                .fillMaxWidth(),
```

Comment:
Make sure to add padding for the navigation bars

Reply:
I put it on the column but I will do the SOP